app/services/setting_registry.rb

```diff
@@ -151,6 +151,7 @@ def known_categories
   def load_values(ignore_cache: false)
     # we are loading only known STIs as we load settings fairly early the first time and plugin classes might not be loaded yet.
     settings = Setting.unscoped.where(category: known_categories)
+    settings = settings.where.not(value: nil)
```

Comment:
Can't these 2 be combined on the same line or even in the same where?

Reply:
one is negated, so it can't be the same where, but it can be on the same line if you feel it would be better :)